src/Language/Haskell/Liquid/GHC/Interface.hs

```diff
@@ -847,7 +847,7 @@ extractSpecQuotes' thisModule getAnns a = mapMaybe extractSpecQuote anns
 
 extractSpecQuote :: AnnPayload -> Maybe BPspec
 extractSpecQuote payload = 
-  case fromSerialized deserializeWithData payload of
+  case Ghc.fromSerialized Ghc.deserializeWithData payload of
```

Comment:
This looks like a change that would be necessary with ghc-810.7 as well.
https://www.stackage.org/haddock/lts-18.17/ghc-lib-parser-8.10.7.20210828/GHC-Serialized.html#v:deserializeWithData